setup.py

```diff
@@ -26,6 +26,7 @@ def read_file(filename, encoding='utf8'):
     'ConfArgParse',
     'jsonschema',
     'mock',
+    'psutil',
```

Comment:
Maybe it's worth to say `'psutil>2.1.0'   # net_connections introduced in 2.1.0: http://pythonhosted.org/psutil/#psutil.net_connections`.

BTW, this does mean, that me might have have to backport newer `python-psutil` to wheezy (cc: @fmarier)